### ISSUE TITLE
No valid module file was found in any module directory?

### ISSUE THREAD

**Issue by Meca213** [jamesstringerparsec/Easy-GPU-PV#63]
whenever i try to run the command on powershell ise as admin i get this error 
22Import-Module : The specified module '\Add-VMGpuPartitionAdapterFiles.psm1' was not loaded because no valid module file was found in any module directory.
At line:22 char:1
+ Import-Module $PSSCriptRoot\Add-VMGpuPartitionAdapterFiles.psm1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (\Add-VMGpuPartitionAdapterFiles.psm1:String) [Import-Module], FileNotFoundException
    + FullyQualifiedErrorId : Modules_ModuleNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand
and right after it shutdowns

**Comment by jamesstringerparsec** [comment]
Run the files from within the extracted folder. Do not move them.

**Comment by Meca213** [comment]
but that's what i did i never moved any of them every file was in the same file

**Comment by jamesstringerparsec** [comment]
Please run the script again and copy the full output rather than just the error snippet.

**Comment by Meca213** [comment]
Okay! Thanks a lot 
Turns out i was opening the files incorrectly with Powershell Ise i got everything working out but now i get this error 

INFO   : Setting up Parsec to install at boot
INFO   : Dismounting VHDX...
INFO   : Closing Windows image...
INFO   : Closing ISO...
INFO   : Done.
INFO   : Starting and connecting to VM
Start-VM : 'MECAVM' failed to start.
The configuration of the virtual machine 'MECAVM' specifies an upper MMIO region that is above the 
supported physical address range.
'MECAVM' failed to start. (Virtual machine ID 24D0A1E9-81C5-4EAB-8B91-29C57E4E3B5C)
Virtual machine 'MECAVM' cannot be started. The upper MMIO region (0x00000017dffff000) is above 
the supported physical address range. Make sure that the upper MMIO region is below 
0x0000001000000000. (Virtual machine ID 24D0A1E9-81C5-4EAB-8B91-29C57E4E3B5C)
At C:\Users\Meca\Downloads\Easy-GPU-PV-main\Easy-GPU-PV-main\CopyFilesToVM.ps1:4365 char:1
+ Start-VM -Name $params.VMName
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Start-VM], VirtualizationException
    + FullyQualifiedErrorId : InvalidParameter,Microsoft.HyperV.PowerShell.Commands.StartVM

I did a little bit of googling and it said it's because of my cpu but i'm not sure also i'm using a i5 650
